text/metrics/0117-min-max-last-sum-count-aggregator.md

```diff
@@ -0,0 +1,192 @@
+# Replace MinMaxSumCount aggregator with MinMaxLastSumCount aggregator
```

Comment:
Was the decision to put "Last" in the middle intentional? It's a bikeshed, but on first read it could be interpreted as "last sum". Wondering if `MinMaxSumCountLast` was considered?

Reply:
Take away from the Metric SIG is MMLSC is fine.